src/tokens/tokens.yml

```
@@ -17,7 +17,8 @@ imports:
   - ./semantic/object-styles/elevation-shadow.yml
   - ./semantic/object-styles/elevation-surface.yml
   - ./semantic/object-styles/focus-shadow.yml
-
+  - ./semantic/object-styles/border-width.yml
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)